capstone/src/main/webapp/multilinevisualization.js

```diff
@@ -0,0 +1,225 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const d3 = window.d3;
+/**
+ * Setup dummy data for sample visualization
+ */
+const dateArr = [
+  new Date(2020, 2, 1),
+  new Date(2020, 2, 8),
+  new Date(2020, 2, 15),
+  new Date(2020, 2, 22),
+  new Date(2020, 2, 29),
+  new Date(2020, 3, 5),
+  new Date(2020, 3, 12),
+];
+const data = {
+  'restaurantData': [
+    {
+      'restaurantName': 'Wildfire',
+      'clickData': [2, 10, 10, 8, 3, 18, 20],
+    },
+    {
+      'restaurantName': 'Poke Doke',
+      'clickData': [0, 8, 10, 5, 20, 22, 17],
+    },
+    {
+      'restaurantName': 'The Goog Noodle',
+      'clickData': [20, 18, 12, 21, 20, 19, 15],
+    },
+  ],
+  'dates': dateArr,
+};
+const height = 400;
```

Comment:
Is it possible to place all the logic for drawing the visualization in a function?

Reply:
Yes, I put everything in a function, let me know if it looks good!